File: src/main/java/net/sjrx/intellij/plugins/systemdunitfiles/lexer/UnitFileLexerAdapter.java
package net.sjrx.intellij.plugins.systemdunitfiles.lexer;

import com.intellij.lexer.FlexAdapter;
import net.sjrx.intellij.plugins.systemdunitfiles.generated.UnitFileLexer;

public class UnitFileLexerAdapter extends FlexAdapter {

  public UnitFileLexerAdapter() {
    super(new UnitFileLexer(null));
  }

}

File: src/main/kotlin/net/sjrx/intellij/plugins/systemdunitfiles/lexer/UnitFileLexerAdapter.kt
package net.sjrx.intellij.plugins.systemdunitfiles.lexer

import com.intellij.lexer.FlexAdapter
import net.sjrx.intellij.plugins.systemdunitfiles.generated.UnitFileLexer

class UnitFileLexerAdapter : FlexAdapter(UnitFileLexer(null))
